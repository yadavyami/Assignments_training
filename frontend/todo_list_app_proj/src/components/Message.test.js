import React from 'react'
import renderer from 'react-test-renderer'
import Message from './Message'

test('Message snapshot with different text', () => {
    const component = renderer.create(<Message text="Hello, Jest!" />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
})
