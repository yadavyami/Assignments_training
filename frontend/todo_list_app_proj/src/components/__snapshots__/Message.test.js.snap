// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message snapshot with different text 1`] = `
<div>
  Hello, Jest!
</div>
`;
